app/views/user_mailer/unconfirmed_email_instructions.html.erb

```diff
@@ -0,0 +1,41 @@
+  <p class="lead" style="margin-bottom: 1em">
+    <%= t('user_mailer.email_confirmation_instructions.first_sentence.forgot_password') %>
+  </p>
+  <p class="lead">
+    <%= t('user_mailer.email_confirmation_instructions.request_with_diff_email',
+         app_name: APP_NAME) %>
+  </p>
```

Comment:
nit: can we unindent these a bit? they don't appear to be nested

Reply:
Not a nit - this should have not made it into the commit.  I need to ensure my IDE formats on save so silly mistakes like this are avoided.  My bad.